docs/site/content/docs/edge/node-pool.md

```diff
@@ -0,0 +1,180 @@
+
+Example node pool definition for Azure cluster:
+
+```yaml
+name: tkg-azure-wc-np-1
+replicas: 2
+az: 2
+nodeMachineType: Standard_D2s_v3
+labels:
+key1: value1
+key2: value2
+```
+
+### vSphere Configuration
+
+In addition to the required `name`, `replicas`, and `labels` properties above, configuration files for node pools on vSphere can include a `vsphere` block, to define optional properties specific to configuring VMs on vSphere.
+
+Example node pool definition for vSphere cluster:
+
+```sh
+name: tkg-wc-oidc-md-1
+replicas: 4
+labels:
+key1: value1
+key2: value2
+vsphere:
+memoryMiB: 8192
+diskGiB: 64
+numCPUs: 4
+datacenter: dc0
+datastore: iscsi-ds-0
+storagePolicyName: name
+folder: vmFolder
+resourcePool: rp-1
+vcIP: 10.0.0.1
+template: templateName
+cloneMode: clone-mode
+network: network-name
+```
+
+Any values not set in the `vsphere` block inherit from the values in the cluster's first node pool.
+
+## Update Node Pools
+
+If you only need to change the number of nodes in a node pool, use the Tanzu CLI command in [Scale Nodes Only](#scale-nodes) below. If you want to add labels as well, follow the procedure in [Add Labels and Scale Nodes](#update-labels).
+
+**Caution:** With these procedures, do not change existing labels, the availability zone, node instance type (on AWS or Azure), or virtual machine properties (on vSphere) of the node pool. This can have severe negative impacts on running workloads. To change these properties, create a new node pool with these properties and migrate workloads to the new node pool before deleting the original.
+
+### Scale Nodes Only
+
+To change the number of nodes in a node pool, run:
+
+```shell
+tanzu cluster scale CLUSTER-NAME -p NODE-POOL-NAME -w NODE-COUNT
+```
+
+Where:
+
+- `CLUSTER-NAME` is the name of the workload cluster.
+- `NODE-POOL-NAME` is the name of the node pool.
+- `NODE-COUNT` is the number of nodes, as an integer, that belong in this node pool.
+
+### Add Labels and Scale Nodes
+
+You can add labels to a node pool and scale its nodes at the same time through the node pool configuration file.
+
+1. Open the configuration file for the node pool you want to update.
+
+1. If you are increasing or decreasing the number of nodes in this node pool, update the number after `replicas`.
+
+1. If you are adding labels, indent them below `labels`. For example:
+
+    ```yaml
+    labels:
+      key1: value1
+      key2: value2
+    ```
+
+1. Save the node pool configuration file.
+
+1. In a terminal, run:
+
+    ```shell
+    tanzu cluster node-pool set CLUSTER-NAME -f <CONFIG-FILE>
+    ```
+
+    If the `CLUSTER-NAME` in the command and `name` in the configuration file match a node pool in the cluster, this command updates the existing node pool instead of creating a new one.
+
+## Delete Node Pools
+
+To delete a node pool run:
+
+```shell
+tanzu cluster node-pool delete <CLUSTER-NAME>
```

Comment:
Is there a nodepool name required for this command? Or is the only option to delete all nodepools?

Reply:
I think so yes, but asking for confirmation and can go back and changed if needed.
This is how it is in the TKG docs:
https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.5/vmware-tanzu-kubernetes-grid-15/GUID-tanzu-cli-reference.html?hWord=N4IghgNiBcIC5gHYC8CuACAxhVBnOApgE7qID2AJgQLQAOZZE6VEBhIAvkA

https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.5/vmware-tanzu-kubernetes-grid-15/GUID-tanzu-k8s-clusters-node-pool.html?hWord=N4IghgNiBcILJgHZgOYFMAEA5A9gE0wAUccIBnDAMxwCcMARAS0srRrUQBcMA1ODACoBPAA5oyIAL5A#delete-node-pools-9

Reply:
Looks like there is probably a `-n` arg needed for that, but looks fine then.